docs/sources/release-notes/_index.md

```diff
@@ -12,6 +12,7 @@ about deprecations, breaking changes as well as changes that relate to plugin de
 
 - [Release notes for 9.0.0-beta3]({{< relref "release-notes-9-0-0-beta3" >}})
 - [Release notes for 9.0.0-beta2]({{< relref "release-notes-9-0-0-beta2/" >}})
+- [Release notes for 9.0.0-beta2]({{< relref "release-notes-9-0-0-beta2" >}})
```

Comment:
Duplicate, bug with the action. Can you fix manually?

Reply:
@marefr done https://github.com/grafana/grafana/pull/50398/commits/80cbff3adc325a98a3d932e736e903b7cdd96b09